cfripper/rules/__init__.py

```diff
@@ -38,6 +38,7 @@
     SQSQueuePolicyNotPrincipalRule,
     SQSQueuePolicyPublicRule,
 )
+from cfripper.rules.stack_name_matches_regex import StackNameMatchesRegexRule
```

Comment:
unless the regex is planning on changing i would suggest a better name is StackNameAlphanumericRule. if the point is to just demonstrate that the regex can be anything ignore this point.